extra-utils/daemonize/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=daemonize
+PKGDES="Command line utility to run a program as a Unix daemon"
+PKGDEP="glibc"
+PKGSEC=utils
+RECONF=0
+
+ABHOST=noarch
```

Comment:
My apologies, but it seems contradictory that this package depends on `glibc`, while being `noarch`. What is going on?

Reply:
Let me check the details...